### ISSUE TITLE
Ensembling MT-DNN models

### ISSUE THREAD

**Issue by dchang56** [namisan/mt-dnn#56]
Hello!
I'd like to know more of the details of ensembling multiple MT-DNN models.
Specifically for the STS-B task, could you describe the steps you took to get the ensemble? Given the large model sizes, did you load them to the GPUs sequentially and just saved the individual model outputs to be averaged later? I'd appreciate your response!

**Comment by namisan** [comment]
I use the pretrained MT-DNN as the  initialization checkpoint, then use different dropout rates, seeds and grad norm (e.g., 1, 2, 5) to train several models. A last, I average the output of predictions as final output.

**Comment by ruanchaves** [comment]
There are two parameters in this library for grad norm: grad_clipping ( which defaults to 0 ) and global_grad_clipping ( which defaults to 1 ). Which one should be used to test different grad norm possibilities? 

Reading [model.py](https://github.com/namisan/mt-dnn/blob/14f08811c34a46e0051ef9c846a040d765c39435/mt_dnn/model.py) and its associated files, it seems to me that leaving grad_clipping at 0 and then testing different values for global_grad_clipping ( e.g. 1, 2, 5 ) would be a suitable approach. Is that correct?

**Comment by ruanchaves** [comment]
@namisan There are two parameters in this library for grad norm: grad_clipping ( which defaults to 0 ) and global_grad_clipping ( which defaults to 1 ). Which one should be used to test different grad norm possibilities? 

Reading [model.py](https://github.com/namisan/mt-dnn/blob/14f08811c34a46e0051ef9c846a040d765c39435/mt_dnn/model.py) and its associated files, it seems to me that leaving grad_clipping at 0 and then testing different values for global_grad_clipping ( e.g. 1, 2, 5 ) would be a suitable approach. Is that correct?
